thermo/EquationsOfState.ipynb

```diff
@@ -0,0 +1,414 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Non-ideal equations of state \n",
+    "This example demonstrates the comparison between ideal and non-ideal equations of state (EoS) from Cantera. The following equations of state are used to evaluate thermodynamic properties (enthalpy, entropy, etc.) in this example -\n",
+    "1. Ideal-gas EoS from Cantera\n",
+    "2. Non-ideal Redlich-Kwong EoS from Cantera\n",
+    "3. Helmholtz energy EoS from CoolProp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Import required packages (Cantera and CoolProp)\n",
+    "CoolProp is an open-source package that contains a highly-accurate database for thermophysical properties. The thermodynamic properties are obtained using pure and pseudo-pure fluid equations of state implemented for 122 components. More details can be found in a reference -\n",
+    "\n",
+    ">I.H. Bell, J.Wronski, S. Quoilin, V. Lemort, “Pure and Pseudo-pure Fluid Thermophysical Property Evaluation and\n",
+    "             the Open-Source Thermophysical Property Library CoolProp,” Industrial & Engineering Chemistry Research 53 (2014), http://pubs.acs.org/doi/pdf/10.1021/ie4033999"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Running Cantera version: 2.5.0b1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Import Cantera\n",
+    "import cantera as ct\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import time\n",
+    "print('Running Cantera version: ' + ct.__version__)\n",
+    "from CoolProp.CoolProp import PropsSI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Helper functions\n",
+    "This example uses CO$_2$ as the only species. The function 'getThermoCantera' calculates thermodynamic properties based on the thermodynamic state (T, p) of the species using Cantera. Applicable gas-phases are Ideal-gas and Redlich-Kwong. The ideal-gas equation can be stated as\n",
+    "\\begin{equation}\n",
+    "    pv = RT,\n",
+    "\\end{equation}\n",
+    "where $p$, $v$ and $T$ represent thermodynamic pressure, molar volume, and the temperature of the gas-phase. $R$ is the universal gas constant.\n",
+    "\n",
+    "The function 'getThermoCoolProp' utilizes the CoolProp package to evaluate thermodynamic properties based on the gas-phase, thermodynamic state (T, p) for a given fluid. The standard-reference thermodynamic states are different for Cantera and CoolProp, it is necessary to convert these values in an appropriate scale before comparison. Therefore, both functions 'getThermoCantera' and 'getThermoCoolProp' return the thermodynamic values relative to the reference state obtained at 1 Bar, 300 K.\n",
```

Comment:
You can use backticks `` ` `` to have code formatting. Please use that for all function and variable names.

Reply:
Please also be consistent about representing thermodynamic variables in math-mode. Here you have T and p outside of math-mode.

Reply:
Corrected.